src/main/java/com/powsybl/openloadflow/sensi/DcSensitivityAnalysis.java

```diff
@@ -288,12 +289,26 @@ private void createBranchSensitivityValue(LfSensitivityFactor<DcVariableType, Dc
             }
         }
 
+        functionValue = fixZeroReferenceFlow(contingency, functionValue);
+
         double unscaledSensi = unscaleSensitivity(factor, sensitivityValue);
         if (!filterSensitivityValue(unscaledSensi, factor.getVariableType(), factor.getFunctionType(), parameters)) {
             resultWriter.writeSensitivityValue(factor.getIndex(), contingency != null ? contingency.getIndex() : -1, unscaledSensi, unscaleFunction(factor, functionValue));
         }
     }
 
+    /**
```

Comment:
Just for coherence: the term "reference flow" was used in the previous API. Now, in DC sensitivity analysis, even if we have only active power flows as function types, we use in general "reference function".